app/styles/local-navbar.css

```diff
@@ -3,6 +3,7 @@
   display: flex;
   justify-content: center;
   padding: 20px;
+  background-color: #ffffff;
```

Comment:
local navbar doesn't require any background I think. It can be transparent (no background)

Reply:
change the localnavar background to transparent.

Reply:
I meant there is no need to add the background-color.

Reply:
Removing the background color causing this  2 errors.
01 ![image](https://user-images.githubusercontent.com/73482745/192089103-19902bd1-a7ee-499c-a82c-a97ba8e3df53.png)

02 
![image](https://user-images.githubusercontent.com/73482745/192089119-4959d2e1-ec04-41d3-a13d-23f64a014401.png)

Reply:
Ohh ok 🤔, let's move forward with the background color for now. We can investigate this in a new issue.